elements/pfe-tabs/src/pfe-tab-panel.scss

```diff
@@ -26,18 +34,28 @@ $LOCAL-VARIABLES: (
   .container {
     margin: 0;
     width: 100%;
-    padding: #{pfe-local($cssvar: Padding, $region: panel, $fallback: #{pfe-var(container-spacer)} 0)};
-
-    border-top:    #{pfe-local(BorderTop, 0, $region: panel)};
-    border-right:  #{pfe-local(BorderRight, 0, $region: panel)};
-    border-bottom: #{pfe-local(BorderBottom, 0, $region: panel)};
-    border-left:   #{pfe-local(BorderLeft, 0, $region: panel)};
+    background-color: pfe-local(BackgroundColor, $region: panel);
+    border-top:    pfe-local(BorderTop, 0, $region: panel);
```

Comment:
Instead of defining the fallback in the function, I think it's more consistent if we define them in the LOCAL-VARIABLES map at the top.  It makes it easier to maintain and update if they're all in one spot too.

Reply:
+1, updated